apps/settings/js/modules/panel_cache.js

```diff
@@ -23,6 +23,7 @@ define(function(require) {
       LazyLoader.load(['shared/style/action_menu.css',
                        'shared/style/confirm.css',
                        'shared/style/progress_activity.css',
+                       '/shared/elements/gaia-icons/bidi-helper.css',
```

Comment:
I guess I can then remove the one in index.html? I only need it for the download panel, and it won't affect startup perf this way, right?